fasjson/web/resources/users.py

```diff
@@ -44,3 +44,32 @@ def get(self, username):
         if res is None:
             api_v1.abort(404, "User not found", name=username)
         return res
+
+
+@api_v1.route("/search/<search_term>/")
+@api_v1.param("search_term", "The term used to find a user or list of users")
+@api_v1.response(404, "User not found")
+class SearchUsers(Resource):
+    @api_v1.doc("search")
+    @api_v1.expect(page_request_parser)
+    @api_v1.marshal_with(UserModel)
+    def get(self, search_term):
+        """Fetch users given a search term"""
+        args = page_request_parser.parse_args()
+        client = ldap_client()
+        filters = (
+            f"(&(objectClass=fasUser)(!(nsAccountLock=TRUE))(|(username=*{search_term}*)"
```

Comment:
The LDAP attribute for ``username`` is ``uid``.

```suggestion
            f"(&(objectClass=fasUser)(!(nsAccountLock=TRUE))(|(uid=*{search_term}*)"
```